docs/source/data/data_catalog.md

```diff
@@ -461,7 +461,75 @@ airplanes:
 
 In this example, the default `csv` configuration is inserted into `airplanes` and then the `load_args` block is overridden. Normally, that would replace the whole dictionary. In order to extend `load_args`, the defaults for that block are then re-inserted.
 
+## Load multiple dataset with similar configuration using dataset factory patterns
+For catalog entries that share a lot of the same configurations, you could also use the dataset factory pattern introduced in Kedro 0.18.11. This syntax allows you to
+reduce the number of similar catalog entries by matching it to a factory pattern.
 
+Consider the following catalog entries:
+```yaml
+factory_data:
+  type: pandas.CSVDataSet
+  filepath: data/01_raw/factory_data.csv
+
+
+process_data:
+  type: pandas.CSVDataSet
+  filepath: data/01_raw/process_data.csv
+```
+The datasets in this catalog can be generalised to the following pattern:
+```yaml
+"{name}_data":
+  type: pandas.CSVDataSet
+  filepath: data/01_raw/{name}_data.csv
+```
+When `factory_data` or `process_data` is used in your pipeline, it would be matched to the factory pattern above. Two things to keep in mind are that the factory pattern name should be enclosed in
```

Comment:
```suggestion
When `factory_data` or `process_data` is used in your pipeline, it will be matched to the factory pattern above. Two things to keep in mind are that the factory pattern name should be enclosed in
```